README.md

```diff
@@ -6,15 +6,19 @@ The variable read is the daily mean temperature 2m above the ground level, short
 ## For whom?
 If you are starting to work with .nc-files this is a good starting point for you.
```

Comment:
what is a `.nc-file`? 
I can't see it in the repo and maybe it needs some contextualisation

Reply:
Ah I see it is referred to in the `read_tas.py` file.

Reply:
whoever looks at this script, will deal with netCDF files, I was assuming 😅